<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2b2b2b;
    color: #f0f0f0;
    font-size: 14px;
}

QLineEdit {
    background-color: #3c3c3c;
    border: 1px solid #555;
    padding: 5px;
    color: #f0f0f0;
	font-size: 10px;
}

QPushButton {
    background-color: #444;
    border: 1px solid #666;
    padding: 5px;
    color: #fff;
}

QPushButton:hover {
    background-color: #555;
}


#infoLabel {
	background-color:rgb(255, 3, 3);
	padding: 5px;
     border-radius: 25px;
}

QToolButton. QDialogButtonBox {
	background-color: #3c3c3c;
	border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="URLVerticalLayout">
    <item>
     <widget class="QLabel" name="URLLabel">
      <property name="text">
       <string>URL Label Placeholder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="URLInput">
      <property name="text">
       <string>URL_input_placeholder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="passwordVerticalLayout">
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password Label Placeholder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordInput">
      <property name="text">
       <string>password_input_placeholder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>40</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="usernameVerticalLayout">
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username Label Placeholder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameInput">
      <property name="text">
       <string>username_input_placeholder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="optionLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Placeholder</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	
	background-color: rgb(139, 139, 139);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/floppy-disk-solid.svg</normaloff>icons/floppy-disk-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="URLListWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="addURLButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	
	background-color: rgb(139, 139, 139);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/plus-solid.svg</normaloff>icons/plus-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="removeURLButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	
	background-color: rgb(139, 139, 139);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/minus-solid.svg</normaloff>icons/minus-solid.svg</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
